tqdm/_tqdm.py

```diff
@@ -282,6 +285,22 @@ def _decr_instances(cls, instance):
         except KeyError:
             pass
 
+    @classmethod
+    def write(cls, s, file=sys.stdout, end="\n"):
+        """
+        Print a message via tqdm (without overlap with bars)
+        """
+        # Clear all bars
+        for inst in cls._instances:
```

Comment:
Yep you're right, thank's. There wasn't this multi-file management feature before in tqdm when I did the PR. Casper seems to have included your change in #150.